texinfo/org-trello.org

```diff
@@ -636,16 +658,162 @@ and thanks to all those that opened [[https://github.com/org-trello/org-trello/i
+issues similar to yours. If your issue already exists, please describe
+your experiences in the comments, if different, and try the
+advice offered there, if any.
+
+If you don’t find an issue like yours, you should create a new issue.
+
+You should include the following information in your issue entry:
+
+- Your org-trello version
+- Your org-mode version
+- Your emacs version
+- Your system type and OS version
+- How you installed org-trello:
+  - melpa
+  - melpa-stable
+  - el-get
+  - tarball
+  - git clone
+  - other
+- A clear, concise and complete description of the issue. Please do state the obvious.
+- Supporting data:
+  - A copy of any pertinent messages produced by org-trello or Emacs
+  - A copy of buffer data necessary to reproduce the issue. (Please avoid posting
+    private data. Do _not_ post your Trello.com or other login credentials.)
+  - If the issue is with installation, include the =*Compile Log*=
+    buffer contents.
+
+Please use GitHub’s markdown formatting to improve readability.
+
+We may have questions about your issue or question if we don’t
+understand. You can help us to help you better and faster by being
+clear, concise and complete in your issue description.
+
+*** Using =M-x org-trello-bug-report=
+
+You may use the bug-report function of org-trello to print some
+information about org-trello to the *Messages* buffer to copy into the
+issue entry. To print the bug-report data, type “M-x
+org-trello-bug-report” or “C-c o B”. Then open the *Messages* buffer
+and copy the block of data it produced to paste into the issue on
+GitHub. The block of data will resemble:
+
+#+BEGIN_EXAMPLE
+  System information:
+  - system-type: darwin
+  - locale-coding-system: utf-8-unix
+  - emacs-version: GNU Emacs 26.1
+  - org version: 9.2
+  - org-trello version: 0.8.0
+  - org-trello path: ~/.emacs.d/elpa/org-trello-20180331.631/org-trello.el
+  - request-backend: curl
+  - kill-whole-line: nil
+#+END_EXAMPLE
+
+If you type “C-u C-c o B” instead, org-trello will open a new issue on
+GitHub for you to complete.
+
+*** Using =orgtrello-log-trace=
+
+To obtain a more verbose output in the **Message** buffer, you can put
+this snippet of code in a =*scratch*= buffer.
+
+#+BEGIN_SRC emacs-lisp
+  (custom-set-variables '(orgtrello-log-level orgtrello-log-trace))
+#+END_SRC
+
+and evaluate it with M-x eval-buffer. Then, return to your org-trello
+buffer and repeat the conditions that cause the issue. Copy the
+resulting detailed log messages into your bug-report.
+
+** CHEK Mailing List
+:PROPERTIES:
+:ALT_TITLE: Mailing List
+:DESCRIPTION: Mailing List
+:END:
+
+This project has a mailing list at librelist.com for communication
+and community among its users and contributors. You may join the
+list by sending an email to:
+
+- subscribe: emacs.org.trello [AT] librelist.com
+- unsubscribe: emacs.org.trello-unsubscribe [AT] librelist.com
```

Comment:
wow, i completely forgot about this.
it's there indeed, http://librelist.com/browser/emacs.org.trello/